code/modules/projectiles/ammunition/ammo_kits.dm

```diff
@@ -0,0 +1,233 @@
+
+//// .35 ////
+/obj/item/ammo_kit
+	name = "Scrap Ammo Kit"
+	desc = "A somewhat jank looking crafting kit. It has a can of signel use tools,cheap pliers, and a box of bullet making materials."
```

Comment:
*single use tools, cheap pliers and a box of bullets

Reply:
```suggestion
	desc = "A somewhat jank looking crafting kit. It has a can of single-use tools, cheap pliers and a box of bullet making materials."
```